src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRoller.java

```diff
@@ -0,0 +1,26 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.EndEffector.RollerUtils.RollerConstants;
+import edu.greenblitz.robotName.subsystems.arm.Wrist;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class RunRoller extends RollerCommand{
+    @Override
+    public void initialize() {
+        roller.rollIn();
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (!Wrist.getInstance().isObjectInside()){
```

Comment:
and what is the note will get to the shooter? add the other condition as an or condition

Reply:
![image](https://github.com/GreenBlitz/CRRRRESCENDO/assets/94316058/47a3fd97-36d8-47d4-a5dd-65e61c0bc2b8)